H/hsa_rocr/bundled/patches/musl-affinity.patch

```diff
@@ -0,0 +1,55 @@
+commit 2996bbf1b9c340b42738bc37053a76f1a52ebe42
```

Comment:
Any chance of upstreaming the patches? :slightly_smiling_face:

Reply:
https://github.com/RadeonOpenCompute/ROCR-Runtime/pull/143

Reply:
I'm not sure about the legacy symlinks patch since I'm still not fully familiar with why they're there and whether that can impact other ROCm users.